tests/tools/cdb2bind.c

```diff
@@ -262,13 +262,102 @@ void test_03()
     test_close(hndl);
 }
 
+// Test to verify empty and NULL values bind as expected.
+void test_04()
+{
+    cdb2_hndl_tp *hndl = NULL;
+
+
+    /*
+    char c1[] = {0};
+    const char *c2 = "";
+    int row_count;
+    int expected_row_count = 1;
+    const char *drop_table = "DROP TABLE IF EXISTS t1";
+    const char *create_table_cmd =
+        "CREATE TABLE t1 {schema {blob c1 null=yes cstring c2[10] null=yes}}";
+    const char *insert_cmd = "INSERT INTO t1 VALUES (@c1, @c2)";
+    const char *select_empty_cmd =
+        "SELECT COUNT(*) FROM t1 WHERE c1 = x'' AND c2 = ''";
+    const char *select_null_cmd =
+        "SELECT COUNT(*) FROM t1 WHERE c1 IS NULL AND c2 IS NULL";
+        */
+
+    test_open(&hndl, db);
+
+    {
+        int type[2] = {CDB2_INTEGER, CDB2_INTEGER};
+        int a = 11, b = 33;
```

Comment:
Would it be interesting to do:
```c
int a = 11;
short b = 33;
```
instead?  The case of binding two different integers of different sizes seems like it could be worth covering.

Also, does the case where `count=0` warrant a test?